src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CTM_ConvectionGridCompMod.F90

```diff
@@ -13,14 +13,17 @@ MODULE CTM_ConvectionGridCompMod
 ! !USES:
 !
       USE ESMF
-      USE MAPL
-      USE GmiConvectionMethod_mod             ! GMI     Convection component
-      USE GenericConvectionMethod_mod         ! Generic Convection component
+      USE MAPL_Mod
```

Comment:
@JulesKouatchou Is there a reason you went back to `MAPL_Mod`? @tclune and the SI Team moved GEOS to `use MAPL` as the default in GEOS.

Reply:
Note there are other moves to `use MAPL_Mod` in other files in this PR as well

Reply:
- MAPL_Mod can be replaced by MAPL. 
- The changes I do not produce zero-diff because the atmospheric pressure is now calculated instead of being read from file.